internal/tiltfile/skylarkTypes.go

```diff
@@ -97,12 +97,12 @@ func runDockerImageCmd(thread *skylark.Thread, fn *skylark.Builtin, args skylark
 }
 
 func addMount(thread *skylark.Thread, fn *skylark.Builtin, args skylark.Tuple, kwargs []skylark.Tuple) (skylark.Value, error) {
-	var mountPoint string
 	var gitRepo gitRepo
+	var mountPoint string
 	if len(fn.Receiver().(*dockerImage).cmds) > 0 {
 		return nil, errors.New("add mount before run command")
 	}
-	err := skylark.UnpackArgs(fn.Name(), args, kwargs, "mount_point", &mountPoint, "local_git_repo", &gitRepo)
+	err := skylark.UnpackArgs(fn.Name(), args, kwargs, "local_git_repo", &gitRepo, "mount_point", &mountPoint)
```

Comment:
can we name these arguments `src` and `dest`? i think it would be more idiomatic to support
add(src=repo, dest=path)

Reply:
Yeah sounds good to me.